src/query/src/parser.rs

```diff
@@ -67,6 +105,7 @@ impl QueryLanguageParser {
     }
 
     // TODO(ruihang): implement this method when parser is ready.
```

Comment:
@waynexia wdyt is this TODO still useful? (I put mine as a reminder to ask, I'll remove it anyway)

Reply:
Ahh it's outdated. This function used to be hard-coded 🤣 Could you help me to remove it?

Reply:
sure thing 👍